client/src/lib/error-telemetry.js

```diff
@@ -9,22 +9,47 @@ const Tail   = require('tail').Tail;
 const fs     = require('fs');
 const logger = require('winston');
 
-const TARGET = '/tmp/test';
+/**
+ * Default behaviour for the output where to send data to when the watched logging file has
+ * a modification detected.
+ */
+function callErrorTelemetryService(app, text) {
+
+  const api = app.service('errorTelemetry');
+
+  api.create({
```

Comment:
I believe it's helpful to `return` this, which will give the caller the Promise returned by `api.create`